reports/default/json-schema.json

```diff
@@ -58,6 +58,17 @@
             "minLength": 4,
             "maxLength": 128
         },
+        "currency_type": {
+            "type": "string",
+            "pattern": "[A-Z]+",
```

Comment:
`pattern` is required when `enum` is used?

Reply:
`minLength` and `maxLength` also

Reply:
Fix